test/functional/http/post/4.multisignature.advanced.js

```diff
@@ -69,7 +70,7 @@ describe('POST /api/transactions (type 4) register multisignature', () => {
 	describe('transactions processing', () => {
 		describe('signatures property', () => {
 			describe('correctly offline signed transaction', () => {
-				it('using ready should be ok but never confirmed', () => {
+				it('using ready false should be corrected and confirmed', () => {
```

Comment:
Improve description stating if the transaction contains all the signatures or it is just partially signed.